src/Tickets/Commerce/Admin_Tables/V2/Orders.php

```diff
@@ -0,0 +1,646 @@
+<?php
+/**
+ * Orders Table V2
+ *
+ * @since TBD
+ *
+ * @package Tribe\Tickets\Commerce\Admin_Tables\V2
+ */
+
+namespace TEC\Tickets\Commerce\Admin_Tables\V2;
```

Comment:
I would honestly avoid the V2 folder if we can.

Reply:
No problem, ill introduce the change in next PR